File: examples/fib.py
import datetime, time
import sys
sys.setrecursionlimit(10**6)
def fib(n):
    if n < 2:
        return n
    else:
        return fib(n - 1) + fib(n - 2)  

t = datetime.datetime.now()
res = fib(35)
print("OK: " + str(res) if res == 9227465 else "WRONG: " + str(res))
print("time:", (datetime.datetime.now()-t))

File: examples/fib.js
eval(`function fib(n){
    if (n < 2) {
        return n
    } else {
        return fib(n - 1) + fib(n - 2)
    }
}`)

let t = Date.now()
let res = fib(35);
console.log(res == 9227465 ?  "OK: " + res : "WRONG: " + res, "time:", Date.now()-t)